content/provider/eas/calendarsync.js

```diff
@@ -340,22 +369,26 @@ eas.sync.Calendar = {
             wbxml.otag("Categories");
                 for (let i=0; i<categories.length; i++) wbxml.atag("Category", tbSync.encode_utf8(categories[i]));
             wbxml.ctag();
-        } else {
+        }
+        // TODO: Server rejects empty category list for exceptions, how else to erase categories?
+        else if (!isException) {
             wbxml.atag("Categories");
         }
 
         //TP PRIORITY (9=LOW, 5=NORMAL, 1=HIGH) not mapable to EAS
         
         //Organizer
-        if (item.organizer && item.organizer.commonName) wbxml.atag("OrganizerName", item.organizer.commonName);
-        if (item.organizer && item.organizer.id) wbxml.atag("OrganizerEmail",  cal.removeMailTo(item.organizer.id));
+        if (!isException) {
+            if (item.organizer && item.organizer.commonName) wbxml.atag("OrganizerName", item.organizer.commonName);
+            if (item.organizer && item.organizer.id) wbxml.atag("OrganizerEmail",  cal.removeMailTo(item.organizer.id));
+        }
 
         //Attendees
         let TB_responseType = null;
         let countAttendees = {};
         let attendees = item.getAttendees(countAttendees);
         
-        if (countAttendees.value > 0) {
+        if (countAttendees.value > 0 && !(isException && asversion == "2.5")) {
```

Comment:
for better readability, move if !(isException && asversion == "2.5") as outer if.